### PR TITLE
Doc/remove duplicate object chain

### PR DESCRIPTION
**Pull request by ansaboutin**
Removed duplicate object chains and replaced them with shorter aliased names.

### CODE REVIEW

File: tests/integration/test_optislang/test_optislang_integration.py [hunk 1]
```diff
@@ -140,10 +140,9 @@ def test_generate_read_mesh(mixing_elbow_geometry):
     if float(meshing.get_fluent_version()[:-2]) < 23.0:
         # Step 3 Generate mesh from geometry with default workflow settings
         meshing.workflow.InitializeWorkflow(WorkflowType="Watertight Geometry")
-        meshing.workflow.TaskObject["Import Geometry"].Arguments = dict(
-            FileName=mixing_elbow_geometry
-        )
-        meshing.workflow.TaskObject["Import Geometry"].Execute()
+        Geo_import = meshing.workflow.TaskObject["Import Geometry"]
```
Comment (seanpearsonuk) [inline tests/integration/test_optislang/test_optislang_integration.py:143]: `geo_import` is preferred.

**Review by seanpearsonuk — APPROVED** [review]

**Review by mkundu1 — APPROVED** [review]

File: tests/test_fluent_session.py [hunk 2]
```diff
@@ -59,10 +59,10 @@ def print_transcript(transcript):
...
 def test_server_exits_when_session_goes_out_of_scope() -> None:
     def f():
-        session = pyfluent.launch_fluent()
-        _fluent_host_pid = session.connection_properties.fluent_host_pid
-        _cortex_host = session.connection_properties.cortex_host
-        _inside_container = session.connection_properties.inside_container
+        session = pyfluent.launch_fluent().connection_properties
+        _fluent_host_pid = session.fluent_host_pid
+        _cortex_host = session.cortex_host
+        _inside_container = session.inside_container
```
Comment (raph-luc) [inline tests/test_fluent_session.py:65]: I think these aren't good changes, the number of lines or characters are not significantly reduced, and the `session` object is no longer a `session` in this case

I would say either revert these changes, or change `session` to a variable name that indicates this refers to `connection_properties`

File: tests/test_fluent_session.py [hunk 3]
```diff
@@ -81,11 +81,11 @@ def f():
...
-        session = pyfluent.launch_fluent(cleanup_on_exit=False)
-        _fluent_host_pid = session.connection_properties.fluent_host_pid
-        _cortex_host = session.connection_properties.cortex_host
-        _inside_container = session.connection_properties.inside_container
-        _cortex_pwd = session.connection_properties.cortex_pwd
+        session = pyfluent.launch_fluent(cleanup_on_exit=False).connection_properties
+        _fluent_host_pid = session.fluent_host_pid
+        _cortex_host = session.cortex_host
+        _inside_container = session.inside_container
+        _cortex_pwd = session.cortex_pwd
```
Comment (raph-luc) [inline tests/test_fluent_session.py:88]: Same as in previous comment

File: tests/test_fluent_session.py [hunk 4]
```diff
@@ -164,17 +164,15 @@ def test_exit_fluent_when_connected_to_running_fluent(
...
-    ) or isinstance(session.connection_properties.inside_container, Container)
-    assert isinstance(session.connection_properties.fluent_host_pid, int)
+    session = new_solver_session.connection_properties
+    assert isinstance(session.ip, str)
+    assert isinstance(session.port, int)
+    assert isinstance(session.password, str)
+    assert isinstance(session.cortex_pwd, str)
+    assert isinstance(session.cortex_pid, int)
+    assert isinstance(session.cortex_host, str)
+    assert isinstance(session.inside_container, (bool, Container))
```
Comment (raph-luc) [inline tests/test_fluent_session.py:174]: Good change

File: tests/test_fluent_session.py [hunk 5]
```diff
@@ -164,17 +164,15 @@ def test_exit_fluent_when_connected_to_running_fluent(
...
-    assert isinstance(session.connection_properties.port, int)
-    assert isinstance(session.connection_properties.password, str)
-    assert isinstance(session.connection_properties.cortex_pwd, str)
-    assert isinstance(session.connection_properties.cortex_pid, int)
-    assert isinstance(session.connection_properties.cortex_host, str)
-    assert isinstance(
-        session.connection_properties.inside_container, bool
-    ) or isinstance(session.connection_properties.inside_container, Container)
-    assert isinstance(session.connection_properties.fluent_host_pid, int)
+    session = new_solver_session.connection_properties
```
Comment (raph-luc) [inline tests/test_fluent_session.py:167]: Same as in previous comments

File: tests/test_fluent_session.py [hunk 6]
```diff
@@ -202,7 +200,7 @@ def _freeze_fluent(s):
     alive = timeout_loop(
         get_container,
         5.0,
-        args=(session.connection_properties.cortex_host,),
+        args=(session.cortex_host,),
```
Comment (raph-luc) [inline tests/test_fluent_session.py:203]: Is this going to work? I think this was a mistaken search and replace, the root session object should not have details about the `cortex_host`

**Review by raph-luc — CHANGES_REQUESTED** [review]

File: tests/test_fluent_session.py (hunk 2 above)
Comment (ansaboutin) [reply tests/test_fluent_session.py]: I have made all recommended changes. Thank you for the feedback

File: tests/test_reduction.py [hunk 7]
```diff
@@ -53,166 +55,156 @@ def _test_context(solver):
...
         locations=solver.setup.boundary_conditions.velocity_inlet,
     )
     assert val == expr_val
-    solver.setup.named_expressions.pop(key="test_expr_1")
+    s_named_express.pop(key="test_expr_1")
 
 
 def _test_min(solver1, solver2):
+    s1_boundary_conditions = solver1.setup.boundary_conditions
+    s2_boundary_conditions = solver2.setup.boundary_conditions
```
Comment (raph-luc) [inline tests/test_reduction.py:73]: I would say either `boundary_conditions_1` or `solver1_boundary_conditions` would make more sense

**Review by raph-luc — COMMENTED** [review]
Minor suggestions

File: tests/test_solvermode/test_models.py [hunk 8]
```diff
@@ -30,70 +30,49 @@ def test_solver_models(load_mixing_elbow_mesh):
...
-        "enhanced-wall-treatment"
-    )
-    assert (
-        solver_session.setup.models.viscous.near_wall_treatment.wall_treatment()
-        == "enhanced-wall-treatment"
-    )
-    assert solver_session.setup.models.viscous.near_wall_treatment.wall_treatment.allowed_values() == [
+
+    models.viscous.model = "k-epsilon"
+    near_wall = models.viscous.near_wall_treatment
```
[the remark this answers is not in the document]
Comment (ansaboutin) [reply tests/test_solvermode/test_models.py:38]: I was worried that lines like this: `near_wall.wall_treatment = "enhanced-wall-treatment"` would mess up the aliasing. Would `near_wall_treatment = "enhanced-wall-treatment"` override the variable?

[the remark this answers is not in the document]
Comment (raph-luc) [reply tests/test_solvermode/test_models.py:38]: You're right, I hadn't noticed it would be used like that, I think that would mess it up. Better keep it as you suggest then.

**Review by raph-luc — APPROVED** [review]
